content/Reducing_Power_Consumption.md

```diff
@@ -33,7 +33,7 @@ must set vblank_flag to a non-zero value.
   ld   hl,vblank_flag  ;hl=pointer to vblank_flag
   xor  a               ;a=0
  @@wait:               ;wait...
-  halt                 ;suspend CPU - wait for ANY interrupt
+  halt                 ;suspend CPU - wait for ANY enabled interrupt
```

Comment:
While we're at this, can you change the code snippet to RGBASM syntax?

Reply:
I think it's done :+1: